guides/common/modules/proc_registering-capsule-to-satellite-server.adoc

```diff
@@ -16,7 +16,7 @@ For more information on manifests and repositories, see {ContentManagementDocURL
 
 .Proxy and Network Prerequisites
 * The {ProjectServer} base operating system must be able to resolve the host name of the {SmartProxy} base operating system and vice versa.
-* The base operating system on which you want to install {SmartProxyServer} must not be configured to use a proxy to connect to the Red Hat CDN.
+* Ensure there is no HTTP proxy configured for {SmartProxyServer} to connect to the {ProjectServer}.
```

Comment:
To summarize the discussion in this PR: it should rather be a requirement to have HTTPS client certificate authentication work from the `{SmartProxyServer}` to `{ProjectServer}`. Often HTTP proxies, including transparent ones mess this up. I'm not sure how best to phrase this, but the current text doesn't fully cover it IMHO.

Reply:
Addressed.